spec/index.bs

```diff
@@ -77,6 +77,8 @@ spec: WEBGL-1; urlPrefix: https://www.khronos.org/registry/webgl/specs/latest/1.
         text: drawingBufferColorSpace; url: DOM-WebGLRenderingContext-drawingBufferColorSpace
     type: dictionary
         text: WebGLContextAttributes; url: WEBGLCONTEXTATTRIBUTES
+    type: dfn
+        text: WebGL Drawing Buffer; url:2.2
```

Comment:
This URL should work and be slightly more stable (I found it by looking in dev tools)
```suggestion
        text: WebGL Drawing Buffer; url:THE_DRAWING_BUFFER
```